src/main/java/org/session9/homework/schoolgradebook/SchoolGradebookManagement.java

```diff
@@ -0,0 +1,33 @@
+package org.session9.homework.schoolgradebook;
+
+public class SchoolGradebookManagement {
+
+    public static void main(String[] args) {
+
```

Comment:
Please remove empty lines like this, as we agreed on our [Coding guidelines](https://docs.google.com/document/d/1YT_rjqUlZ8cn8ZI7QAD52LXZq8oVD5_Gq0b5uO40fYk/edit?usp=sharing) manual.